packages/connector-chromium/README.md

```diff
@@ -0,0 +1,63 @@
+# Chromium (`@hint/connector-chromium`)
+
+A connector to use Chromium based browsers via
+[puppeteer][puppeteer] in `webhint`.
+
+## Installation
+
+First, you need to install [`webhint`](https://webhint.io/):
+
+```bash
+npm install hint
+```
+
+Then, install the new connector:
+
+```bash
+npm install @hint/connector-chrome
+```
+
+## Usage
+
+Configure the connector name in your [`.hintrc`][hintrc]
+configuration file:
+
+```json
+{
+    "connector": {
+        "name": "chromium"
+    },
+    ...
+}
+```
+
+## Options
+
+The set of settings supported by Chrome connector are:
+
+```json
+{
+    "connector": {
+        "name": "chromium",
+        "options": {
+            "defaultProfile": true,
+            "useTabUrl": false,
+            "tabUrl": "https://empty.webhint.io/",
+            "flags": ["--headless", "--disable-gpu"],
+            "waitForContentLoaded": 10000
```

Comment:
Random thought: once we have support for actions, should one of the scenarios be allowing the caller to tell us when they're done? E.g. I could see wanting to use `puppeteer`'s `waitForSelector` to keep the scan alive until a particular element has been added to the page. Seems like maybe we discussed this before, but I don't recall for sure...

Reply:
Don't care too much about the `README.md` it's a copy paste of the chrome one with just a few tweaks. Needs to be reviewed entirely once we lock the options and such

Reply:
> Seems like maybe we discussed this before, but I don't recall for sure...

We had some discussions about custom `options` and what would the api be for the user. We can discuss that when I start the actions implementation. The biggest thing to take into account is how to start monitoring for things. Maybe the user will have to tell us explicitely in some scenarios.